app/components/ui/atoms/Textarea.js

```diff
@@ -45,14 +41,24 @@ class Textarea extends React.Component {
     this.inputId = `textarea-${Math.round(Math.random() * 1e12).toString(36)}`
   }
   render() {
-    const { label, value = '', readonly, ...props } = this.props
+    const { label, ...props } = this.props
     return (
-      <Root>
-        {label && <Label htmlFor={this.inputId}>{label}</Label>}
-        <Input id={this.inputId} readOnly={readonly} value={value} {...props} />
```

Comment:
Javascript seems to be case sensitive with variables names, are you sure this works?